src/utils.js

```diff
@@ -42,7 +42,13 @@ export function get<T: any>(uri: string, options: any): Promise<T> {
             return;
         }
 
-        const opts = Object.assign({ json: true }, options);
+        const opts = Object.assign(
+            {
+                json: false,
+                token: null,
```

Comment:
May need to decouple this from the `get()` since other future providers may not follow the same authentication format.